crates/miden-lib/asm/kernels/transaction/lib/output_note.masm

```diff
@@ -24,14 +24,6 @@ const.ERR_OUTPUT_NOTE_INDEX_OUT_OF_BOUNDS="requested output note index should be
 #! Panics if:
 #! - the note index is greater or equal to the overall number of output notes.
```

Comment:
This is no longer the case, right? But we should probably add a comment explaining that we are expecting the index to be in bounds.